src/module/system/damage/weapon.ts

```diff
@@ -221,38 +223,6 @@ class WeaponDamagePF2e {
         // Potency rune
         const potency = weaponPotency?.bonus ?? 0;
 
-        // Striking rune
-
-        const strikingSynthetic = domains
-            .flatMap((key) => actor.synthetics.striking[key] ?? [])
-            .filter((wp) => wp.predicate.test(options))
-            .reduce(
-                (highest: StrikingSynthetic | null, current) =>
-                    highest && highest.bonus > current.bonus ? highest : current,
-                null,
-            );
-        // Add damage dice if the "weapon" is an NPC attack or actual weapon with inferior etched striking rune
-        if (
-            strikingSynthetic &&
-            baseDamage.die &&
-            (weapon.isOfType("melee") || strikingSynthetic.bonus > weapon.system.runes.striking)
-        ) {
-            damageDice.push(
-                new DamageDicePF2e({
-                    selector: `${weapon.id}-damage`,
-                    slug: "striking",
-                    label: strikingSynthetic.label,
-                    diceNumber: strikingSynthetic.bonus,
-                }),
-            );
-
-            // Remove extra dice from weapon's etched striking rune
-            if (weapon.isOfType("weapon")) {
-                weapon.system.damage.dice -= weapon.system.runes.striking;
-                weapon.system.runes.striking = 0;
-            }
-        }
-
         // Critical specialization effects
         const critSpecEffect = ((): CritSpecEffect => {
```

Comment:
What of crit spec effects that depend on number of damage dice?

Reply:
It seems to work perfectly. The crit specs are collected, but they're not computed yet. We can move the ordering if that makes you feel comfortable with regards to potential future breakage though.